cmd/imagestream.go

```diff
@@ -35,14 +37,16 @@ func NewImageStreamCleanupCommand() *cobra.Command {
 	cmd.Flags().StringVarP(&o.RepoPath, "git-repo-path", "p", ".", "absolute path to Git repository (for current dir use: $PWD)")
 	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "", "Kubernetes namespace")
 	cmd.Flags().IntVarP(&o.Keep, "keep", "k", 10, "keep most current <n> images")
+	cmd.Flags().BoolVarP(&o.Tag, "tag", "t", false, "use tags instead of commit hashes")
+	cmd.Flags().StringVar(&o.Sorted, "sort-by", git.Version.String(), "sort tags either by version or in alphabetical order. The sort can be version or alphabetic")
```

Comment:
I would suggest "sort tags by criteria. Allowed values: [version, alphabetical]". Also rename to "sort" only, so that the flag "--sort alphabetical" makes more sense than "--sort-by alphabetical"